src/Services/Qrda/Util/DateHelper.php

```diff
@@ -20,7 +20,7 @@ class DateHelper
      */
     public static function format_datetime_cqm($datetime)
     {
-        return gmdate('Y-m-d\TH:i:s.u', date('U', strtotime($datetime)));
+        return !empty($datetime) ? date('Y-m-d\TH:i:s.u', strtotime($datetime)) : null;
```

Comment:
@kchapple you may want to do this differently but it's important we keep local same as the imported datetimes.
I get to bumping the cqm execution module dependency tomorrow...